tailwind.config.js

```diff
@@ -0,0 +1,598 @@
+const plugin = require( 'tailwindcss/plugin' );
+
+module.exports = {
+	purge: {
+		content: ['./**/*.php'],
+		layers: ['utilities'],
+		mode: 'layers',
+	},
+	theme: {
+		extend: {
+			maxHeight: {
+				'0': '0',
+			},
+		},
+		screens: {
+			'phone': '300px',
+			'tablet-portrait': '600px',
+			'wp-admin-bar': '783px',
+			'tablet-landscape': '900px',
+			'desktop-min': {'min': '1200px'},
+			'desktop': '1200px',
+			'desktop-large': '1600px',
+		},
+		container: {
+			center: true,
+			screens: {
+				'phone': '100%',
+				'desktop': '1200px',
+			},
+		},
+	},
+	future: {
+		purgeLayersByDefault: true,
+	},
+	variants: {},
+	plugins: [
+		plugin( function({ addBase, config }) {
```

Comment:
It might be worth extracting these plugins into separate files to reduce the size of the main config file.